CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ For more information about this file see also [Keep a Changelog](http://keepacha
 - Cleanup of leap year logic, using new `PEcAn.utils::days_in_year(year)` function (#801).
 - Replace many hard-coded unit conversions with `udunits2::ud.convert` for consistency, readability, and clarity
 - Added a new retry.func() to base/utils to provide ability to re-try a function X times before stopping.  Currently using this function in the download.CRUNCEP() function to handle slow responses from THREDDS.
+-Reformatted call_MODIS netcdf output
```

Comment:
In the future, I'd recommend a bit more detail -- you're not saying anything about how it was reformatted